composer.json

```diff
@@ -52,7 +52,7 @@
         "hashids/hashids": "^4.1"
     },
     "require-dev": {
-        "contao/core-bundle": "4.9.*",
+        "contao/core-bundle": "4.13.*",
```

Comment:
You can remove this line. It forced the dev-version of Isotope to install 4.9 instead of 4.13, because the requirement would allow for that. But since 4.13 is the last release, we don't need it anymore 🙃